extensions/SQLServer/SQLServer/SqlServerConfig.cs

```diff
@@ -61,4 +61,9 @@ public class SqlServerConfig
     /// The SQL Server tags table name.
     /// </summary>
     public string TagsTableName { get; set; } = DefaultTagsTableName;
+
+    /// <summary>
+    /// It tells if we should use the vector search or not.
```

Comment:
could you add a link to the relevant feature docs?

Reply:
The feature is currently in Early Adopter Preview, so the only useful link is this one: https://devblogs.microsoft.com/azure-sql/announcing-eap-native-vector-support-in-azure-sql-database. There isn't an official documentation page at the moment. Should we use this link, and update it with new resources when available?

Reply:
yes the link would be useful :-)